arachni/redis.run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -eu
+
+exec redis-server /etc/redis.conf
```

Comment:
From the included `redis.conf`, which counts 1050 lines, it is very difficult to tell if you're using any mechanisms to drop root privileges. Since redis doesn't need such privileges to bind its port (6379<1024), it would be preferable to drop them right here.